### ISSUE TITLE
Warning: Unknown prop `classSet` on <input> tag. Remove this prop from the element

### ISSUE THREAD

**Issue by m-lautenbach** [uber/r-dom#36]
I receive a warning when using classSet.
From a look at the code, it looks like classSet is mapped to the classnames property, but never removed from the properties and subsequentially treated like an unknown property.
Minimal code example:
```
        r.input
            classSet:
                'input-inline': true

```

**Comment by ChristophVitz-Actano** [comment]
Same here, since `react-15.2.0`.

**Comment by mlmorg** [comment]
Should be fixed in `v2.2.3`. Thanks for the heads up.
